tests/test_sidecar2cdxj.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+
+import surt
+from unittest.mock import patch
+from warcio.timeutils import iso_date_to_timestamp
+
+import sidecar2cdxj
+
+
+TEST_DIR = os.path.dirname(__file__)
+
+TEXT_META_FILE = os.path.join(TEST_DIR, 'text.warc.meta.gz')
+
+WARC_PAYLOAD = json.dumps({'Identified-Payload-Type': {'fido': 'text/html',
+                                                       'python-magic': 'text/html'},
+                           'Preservation-Identifier': 'fmt/471',
+                           'Charset-Detected': {'encoding': 'utf-8', 'confidence': 0.99},
+                           'Languages-cld2': {'reliable': True,
+                                              'text-bytes': 7896,
+                                              'languages': [{'name': 'ENGLISH',
+                                                             'code': 'en',
+                                                             'text-covered': 99,
+                                                             'score': 1070.0}]}})
+
+
+def test_surt():
+    surt_url = surt.surt('https://www.example.com/example_arg')
```

Comment:
I don't really think we need to have a `test_surt` and `test_isodate_to_timestamp`, since that is testing third party code that you didn't write. But if you prefer to keep them in, I am ok with that.